src/rules/__tests__/no-jest-import.test.ts

```diff
@@ -0,0 +1,86 @@
+import { RuleTester as ESLintRuleTester } from 'eslint';
+import { TSESLint } from '@typescript-eslint/experimental-utils';
+import rule from '../no-jest-import';
+
+const RuleTester: TSESLint.RuleTester = ESLintRuleTester as any;
+
+const ruleTester = new RuleTester({
+  parser: '@typescript-eslint/parser',
+  parserOptions: {
+    ecmaVersion: 6,
+  },
+});
+
+const moduleSourceType = { sourceType: 'module' } as const;
```

Comment:
can ditch all `as const` after https://github.com/typescript-eslint/typescript-eslint/pull/517 is released